README.md

```diff
@@ -209,6 +209,25 @@ To run black:
 black .
 ```
 
+### Run migration Scripts
+
+* For new contributors who just completed environment setup, run the following command **AFTER** running the application for the first time.
+
+```
+flask db stamp head
+```
+
+This will attach the alembic version number on the repo (if any) to the local db.
+
+* For existing contributors, run the following command:
```

Comment:
Here existing contributors means whose DB is filled with some entries or those who run `flask db upgrade` in the past.

Reply:
Those whose db is filled with the previous version.

Reply:
@devkapilbansal and @epicadk , I might have to make it clearer here to distinct the 3 situations that might apply to contributors whose dealing with the migration (as I mentioned in [this](https://github.com/anitab-org/mentorship-backend/pull/1038#issuecomment-831534613) post). Please do share your opinion 😉 .

Reply:
Sure @mtreacy002 
It would be good to make the documentation as clear as possible.

Reply:
Ok, @devkapilbansal , should I add those 3 steps on this PR or on #1038 ?

Reply:
I don't think that #1038 is required now. Since we are going to merge this branch in develop once completed